extensions/amp-story-interactive/0.1/amp-story-interactive-img.css

```diff
@@ -13,3 +13,58 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-story-interactive-img-container {
+  --i-amphtml-interactive-img-option-size: 7.5em !important;
+  --i-amphtml-interactive-img-option-margin: 1em !important;
+
+  border-radius: 3.75em 3.75em 4.375em 4.375em !important;
+}
+
+.i-amphtml-story-interactive-img-option-container {
+  background: var(
+    --i-amphtml-interactive-options-chip-background-color
+  ) !important;
+  display: flex !important;
+  justify-content: center !important;
+  border-radius: 4.375em !important;
+  overflow: hidden !important;
+  box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
```

Comment:
`box-shadow` can be removed for now and added in a PR that integrates chip-style="shadow".